internal/test/envtest/external.go

```diff
@@ -0,0 +1,53 @@
+package envtest
+
+import (
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func ExternalCRDDirectoryPaths() []string {
```

Comment:
We already have something similar in https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/blob/main/test/helpers/envtest.go#L162 - can we reuse this instead?